flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/operators/rank/window/WindowRankOperatorBuilder.java

```diff
@@ -125,7 +125,7 @@ public WindowRankOperatorBuilder windowEndIndex(int windowEndIndex) {
                 rankStart > 0,
                 String.format("Illegal rank start %s, it should be positive!", rankStart));
         checkArgument(
-                rankEnd > 1,
+                rankEnd >= 1,
                 String.format("Illegal rank end %s, it should be bigger than 1!", rankEnd));
```

Comment:
```suggestion
                String.format("Illegal rank end %s, it should be at least 1!", rankEnd));
```

Reply:
good catch. I've updated the exception message. Thanks a lot.